tests/integration_tests/analytics/test_get.py

```diff
@@ -87,13 +87,35 @@ def test_get_analytics_summary_unfccc(
     assert list(data.keys()) == EXPECTED_ANALYTICS_SUMMARY_KEYS
 
     assert dict(sorted(data.items())) == {
-        "n_collections": 0,
+        "n_collections": 1,
         "n_documents": 2,
-        "n_events": 3,
+        "n_events": 1,
         "n_families": 1,
     }
 
 
+def test_get_analytics_summary_other(
+    client: TestClient, data_db: Session, another_org_user_header_token
+):
+    setup_db(data_db)
+    response = client.get(
+        "/api/v1/analytics/summary",
+        headers=another_org_user_header_token,
+    )
+    assert response.status_code == status.HTTP_200_OK
+
+    data = response.json()
+    assert isinstance(data, dict)
+    assert list(data.keys()) == EXPECTED_ANALYTICS_SUMMARY_KEYS
+
+    assert dict(sorted(data.items())) == {
+        "n_collections": 1,
```

Comment:
Is the collection returned here an empty one? ... Just surprised the user can see a single collection and no families :thinking:

Reply:
Ah yes - just seen -great :+1:

Reply:
Yes it is!